source/production/F0.Minesweeper.Logic/Mineplacer/IMineplacer.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic.Mineplacer
+{
+	internal interface IMineplacer
```

Comment:
I think `minelayer` is the "technical" term

Reply:
Renamed all occurences of `Mineplacer`